src/simpleocv.cpp

```diff
@@ -205,9 +332,11 @@ bool imwrite(const std::string& path, const Mat& m, const std::vector<int>& para
     else
     {
         // unknown extension type
+        img.release();
```

Comment:
Possible memory leak since `m` was cloned into `img`. I released `img` before each return.

Reply:
release is not needed. All the objects on stack will be released automatically on scope ends